src/timepicker/__tests__/time-picker.scenario.js

```diff
@@ -22,26 +22,28 @@ const overrides = {
 };
 
 const Controlled = ({
-  format,
-  step,
   size = 'default',
   initialDate,
   creatable = false,
+  onChange = () => {},
+  ...restProps
 }) => {
   const [time, setTime] = useState(initialDate);
   return (
     <React.Fragment>
       <TimePicker
-        format={format}
-        step={step}
         value={time}
-        onChange={setTime}
+        onChange={time => {
+          setTime(time);
+          onChange();
```

Comment:
what does this do?

Reply:
Since it's a reusable component so I added it for testing with different console logs on separate instances. Just kept it there with a default noop.